### ISSUE TITLE
Problem the loss training from scratch on each epochs

### ISSUE THREAD

**Issue by huyle73** [lindawangg/COVID-Net#220]
Hi guys, I have a problem on training from scratch with 10.000 images for training dataset and 2.000 images for validation dataset. The loss increases rapidly on each epoch. Can you tell me how to reduce the loss on training step? 

Description on image:
![Screen Shot 2022-03-31 at 11 33 57 PM](https://user-images.githubusercontent.com/54856997/161105615-31792eb8-f599-4a0c-b8a2-0bad44d0e21e.png

Hope you can reply me soon!

Best regards,
Le Huy
